esx_service/vmdk_ops.py

```diff
@@ -1491,8 +1491,8 @@ def execRequestThread(client_socket, cartel, request):
             send_vmci_reply(client_socket, reply_string)
         else:
             logging.debug("execRequestThread: req=%s", req)
-            # If req from client does not include version number, set the version to "1" by default
-            client_protocol_version = int(req["version"]) if "version" in req else 1
+            # If req from client does not include version number, set the version to "2" by default
```

Comment:
"2" seems a magic number here. Shouldn't we just use the constant "SERVER_PROTOCOL_VERSION"?